settings.gradle.kts

```diff
@@ -32,3 +32,12 @@ include(
     "sentry-samples:sentry-samples-spring",
     "sentry-samples:sentry-samples-spring-boot",
     "sentry-samples:sentry-samples-spring-boot-webflux")
+
+gradle.beforeProject {
```

Comment:
before we were executing `outputs.dir("${project.projectDir}/$sentryNativeSrc")` so the task was cached and not executed every time, does it mean that now every project sync would try to init all the sub modules?

Reply:
this line wasn't really doing anything, because the submodules paths are dictated by the `.gitmodules` file, and there's no "output" when executing `git submodule ...` per se.

When the modules are already initialized, doing a project sync wouldn't trigger another init of submodules (the git command would just be ignored), unless the submodule was updated to the latest commit - but that's what we want then, to have this automatically bootstrapped/updated in case there are updates.

Reply:
sounds good, lets merge it